disablestrongnamevalidation.ps1

```diff
@@ -0,0 +1,2 @@
+& "C:\Program Files (x86)\Microsoft SDKs\Windows\v10.0A\bin\NETFX 4.6.2 Tools\x64\sn.exe" -Vr *,31bf3856ad364e35
+& "C:\Program Files (x86)\Microsoft SDKs\Windows\v10.0A\bin\NETFX 4.6.2 Tools\sn.exe" -Vr *,31bf3856ad364e35
```

Comment:
`sn -Vr` writes to the registry. The 64-bit tool writes to
>HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\StrongName\Verification

Whereas the 32-bit tool writes to
>HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\Microsoft\StrongName\Verification

(or rather, it writes to the same location, but on 64-bit Windows, WOW64 redirection kicks in)

These registry keys are read by the CLR loader (fusion) at runtime and, again, the bitness of the calling process determines which one is read.

Therefore, you'll need both if you want this to work for both 32-bit and 64-bit processes.